packages/synapse-react-client/src/style/components/_expandable_content.scss

```diff
@@ -22,6 +22,7 @@ $gutter: 15px;
       align-items: center;
       word-break: keep-all;
       padding: 0px;
+      text-transform: capitalize;
```

Comment:
Remove the bootstrap-4-backport class from UserCardListGroups (which breaks MUI button styling), but maintain the text capitalization in Mobile button spans